nsxt/data_source_nsxt_management_cluster.go

```diff
@@ -30,42 +30,31 @@ func dataSourceNsxtManagementCluster() *schema.Resource {
 }
 
 func dataSourceNsxtManagementClusterRead(d *schema.ResourceData, m interface{}) error {
-	nsxClient := m.(nsxtClients).NsxtClient
-	if nsxClient == nil {
-		return dataSourceNotSupportedError()
-	}
+	connector := getPolicyConnector(m)
+	client := nsx.NewClusterClient(connector)
 
-	clusterObj, resp, err := nsxClient.NsxComponentAdministrationApi.ReadClusterConfig(nsxClient.Context)
+	hostIP, err := net.LookupIP(m.(nsxtClients).Host[len("https://"):])
```

Comment:
Shouldn't we skip this if host is already IP address? Can we reuse `resolveHostIPs` function here?

Reply:
net.LookupIP does that exactly[already](https://github.com/golang/go/blob/master/src/net/lookup.go#L309), no reason to double check IMO.